openslides_backend/migrations/migrations/0067_rename_guest_field.py

```diff
@@ -0,0 +1,17 @@
+from datastore.migrations import RenameFieldMigration
+
+
+class Migration(RenameFieldMigration):
```

Comment:
The RenameFieldMigration is based on the old EventMigration system that isn't supposed to be used anymore. I think this will fail in production at the latest. Please write this Migration based on the BaseModelMigration class